src/Apps/Auction/Routes/Register/index.tsx

```diff
@@ -109,16 +148,28 @@ export const RegisterRoute: React.FC<RegisterProps> = props => {
 
     createCreditCard(token.id)
       .then(() => {
-        createBidder().then(() => {
+        createBidder().then((data: RegisterCreateBidderMutationResponse) => {
           setSubmitting(false)
 
+          trackRegistrationSuccess(data.createBidder.bidder.id)
+
           window.location.href = `${sd.APP_URL}/auction/${
             sale.id
           }/confirm-registration`
         })
       })
       .catch(error => {
         logger.error(error)
+        let errorMessages: string[]
+        if (Array.isArray(error)) {
+          errorMessages = error.map(e => e.message)
+        } else if (typeof error === "string") {
+          errorMessages = [error]
+        } else if (error.message) {
+          errorMessages = [error.message]
+        }
```

Comment:
In all cases we've tested the final `error.message` case applied here (though at least a `PayloadError[]` is a possible outcome), so this is just being pessimistic.